WalletWasabi.Fluent/ViewModels/Dialogs/Authorization/AuthorizationDialogBase.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Reactive.Linq;
+using System.Threading.Tasks;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Transactions;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+
+namespace WalletWasabi.Fluent.ViewModels.Dialogs.Authorization
+{
+	public abstract class AuthorizationDialogBase : DialogViewModelBase<bool>
+	{
+		protected AuthorizationDialogBase()
+		{
+			var canExecute = this.WhenAnyValue(x => x.IsDialogOpen).ObserveOn(RxApp.MainThreadScheduler);
```

Comment:
ObserveOn isnt required, IsDialogOpen never touched on anything by the ui thread...

Is this canExecute even needed? User cannot click the button if the dialog isnt open?

Reply:
It was there before. I will investigate if needed or not

Reply:
seemingly it is not needed at all, I delete the canExecute checks.